docs/reporters/AsciiDocTemplateReporter.md

```diff
@@ -36,8 +36,13 @@ cli/build/install/ort/bin/ort report
   --report-option AsciiDocTemplate=pdf-theme.path=pdf-theme.yml
 ```
 
+If you want to add your own custom fonts you need to add the directory in which the fonts are with the additional 
+`reporter-option` `--report-option AsciiDocTemplate=pdf-fontsdir=path/to/fonts/` if you want to use a [relative 
```

Comment:
I don't like the way how both `reporter-option` and `--reporter-option` are mentioned here. How about this:

> If you want to use your own custom fonts you need to add the directory in which the fonts are located as a report-specific option like `--report-option AsciiDocTemplate=pdf-fontsdir=path/to/fonts/` where `path/to/fonts/` is the relative path to the font directory from the ... directory.

Above, you still need to replace "..." with the base directory to which the font directory is relative to.